central/processlisteningonport/store/postgres/full_store.go

```diff
@@ -114,6 +114,8 @@ func (s *fullStoreImpl) readRows(
 			return nil, err
 		}
 
+		podUID = msg.GetPodUid()
```

Comment:
This looks strange. PodUID is a normal column now, but we still extract the value out of the serialized blob, why?

Reply:
It does look strange.  The way our database is currently used, columns are mostly for filtering or direct selects.  The serialized field contains the entire snapshot.  So if we pull the serialized field using that is consistent with how the generated stores work.

Reply:
Well removing it from serialized would be a huge re-design.  We put it in a column if we need it for performance so we don't have to walk each object.  As as is we need some fields in both serialized and column.  Some of this was driven because the unmarshaling performance of jsonb was significantly worse than the current design.